packages/ui/tailwind.config.js

```diff
@@ -277,6 +277,7 @@ export default {
       sidebar: "0px 14px 42px 0px rgba(0, 0, 0, 0.08)",
       "app-sidebar": "0px 14px 42px 0px rgba(0, 0, 0, 0.08)",
       "app-navbar": "0px 4px 20px 0px rgba(0, 0, 0, 0.08)",
+      "mobile-app-navbar": "0px 4px 20px 0px rgba(0, 0, 0, 0.1)",
```

Comment:
Specificity should be at the end. 

```suggestion
      "app-navbar-mobile": "0px 4px 20px 0px rgba(0, 0, 0, 0.1)",
```